src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -28,19 +28,24 @@
 import org.junit.Test;
 
 import com.puppycrawl.tools.checkstyle.api.AuditEvent;
+import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
 
 public class DefaultLoggerTest {
 
     @Test
     public void testCtor() throws UnsupportedEncodingException {
+        final LocalizedMessage addExceptionMessage = new LocalizedMessage(0,
+                Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.addException",
+                new String[] {"myfile"}, null,
+                LocalizedMessage.class, null);
```

Comment:
Also class defined in 2nd to last parameter should be the class creating the message, not the message class itself.
Example: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/Checker.java#L323